tests/unit_tests/c_wrappers/res/enkf/test_res_config.py

```diff
@@ -514,7 +512,7 @@ def test_res_config_dict_constructor(setup_case):
         ert_share_path + "/workflows/jobs/internal-gui/config",
     ]
     for ip in config_data_new[ConfigKeys.INSTALL_JOB]:
-        ip[ConfigKeys.PATH] = os.path.realpath(ip[ConfigKeys.PATH])
+        ip[1] = os.path.realpath(ip[1])
```

Comment:
Related to the error in tests what does this actually do for the test? Seems to me it can be removed

Reply:
I can attempt it, but it should disambiguate between relative path and absolute path.

Reply:
Ok, I will remove it then.

Reply:
Removed.